traitsui/qt4/extra/checkbox_renderer.py

```diff
@@ -93,7 +93,11 @@ def paint(self, painter, option, index):
             box.rect.setLeft(option.rect.left() + option.rect.width() // 2 -
                              size.width() // 2 + margin - 2)
 
-        box.state = QtGui.QStyle.State_Enabled
+        # We mark the checkbox always active even when not selected, so
+        # it's clear if it's ticked or not on OSX. See bug #439
+        if option.state & QtGui.QStyle.State_Enabled:
```

Comment:
Sanity check: you do mean bitwise and here?

Reply:
Yes, it's a bit field.